test/promised-sleep.ts

```diff
@@ -0,0 +1,3 @@
+export function promisedSleep(ms: number): Promise<void> {
```

Comment:
here it is

Reply:
At some point we should pull this out to its own module. I think I've got a copy of this in every repo I've made 😅